### PR TITLE
vim: update Spanish translation

### PR DESCRIPTION
**Pull request by victorhck**
updated to sync after PR #6237

- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Converting this to draft, for now.

**Comment by victorhck** [comment]
sorry, I thought that the PR was merged...

**Comment by marchersimon** [comment]
Not yet unfortunately

**Review by marchersimon — APPROVED** [review]
This looks like it's already synced with #6237 
Thanks @victorhck

**Comment by victorhck** [comment]
Great, I was a few days off... :)

File: pages.es/common/vim.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # vim
 
 > Vim (Vi IMproved), un editor de texto para la línea de comandos, que proporciona varios modos para diferentes tipos de manipulación de texto.
-> Pulsando `i` entra en el modo insertar. `<Esc>` habilita el uso de comandos de Vim.
+> Pulsando `i` entra en el modo insertar. `<Esc>` regresa al modo normal, lo que no permite insertar texto.
```
Comment (navarroaxel) [inline pages.es/common/vim.md:4]: I think the main page try to highlight which mode enables users to enter commands.
```suggestion
> Pulsando `i` entra en el modo insertar. `<Esc>` regresa al modo normal, permitiendo el uso de comandos Vim.
```

**Comment by victorhck** [comment]
after many changes I was a bit confusing... That's finally the text merged in English version.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**